adr/0023-mid-cycle-reporting.md

```diff
@@ -0,0 +1,28 @@
+# 23. Mid-cycle Reporting
+
+**Date:** 19/04/2024
+
+## Status:
+
+Proposed
+
+
+## Context
+
+Last cycle we created a static “Mid-cycle Report”, viewable by Provider users in Manage, based on an uploaded CSV file (done only once). See Design History: Letting training providers view their recruitment performance.
+
```

Comment:
Given you mention Monthly Reporting below, I think it would be worth including a note here about what that is and how it is different from the per-provider / reporting.